src/nupic/frameworks/opf/common_models/cluster_params.py

```diff
@@ -86,10 +87,14 @@ def getScalarMetricWithTimeOfDayAnomalyParams(metricData,
     maxVal = minVal + 1
 
   # Load model parameters and update encoder params
-  paramFileRelativePath = os.path.join(
-    "anomaly_params_random_encoder",
-    "best_single_metric_anomaly_params.json")
-
+  if (detectorType is "numenta"):
+    paramFileRelativePath = os.path.join(
+      "anomaly_params_random_encoder",
+      "best_single_metric_anomaly_params.json")
+  elif (detectorType is "numentaNew"):
+    paramFileRelativePath = os.path.join(
+      "anomaly_params_random_encoder",
+      "best_single_metric_anomaly_params_new.json")
```

Comment:
"new" can be confusing outside the context of this PR. It would be better to name the original json "best_single_metric_anomaly_params_cpp.json" and the new json "best_single_metric_anomaly_params_tm_cpp.json"